Scripted/CIP_ParenchymaAnalysis/CIP_ParenchymaAnalysis.py

```diff
@@ -329,6 +348,43 @@ def onCTSelect(self, node):
             for color in ['Red', 'Yellow', 'Green']:
                 slicer.app.layoutManager().sliceWidget(color).sliceLogic().GetSliceCompositeNode().SetBackgroundVolumeID('None')
 
+    def onSegmentationSelect(self, node):
+
+        self.segmentationNode = node
+        if self.segmentationNode: 
+            if self.segmentationNode.GetSegmentation().GetNthSegmentID(0) != "right lung" or \
+                self.segmentationNode.GetSegmentation().GetNthSegmentID(1) != "left lung" or \
+                    self.segmentationNode.GetSegmentation().GetNthSegmentID(2) != "other":
+                qt.QMessageBox.warning(slicer.util.mainWindow(),
+                                       "Parenchyma Analysis", "Input segmentation not valid.")
+                return False
+            else: 
+                segmentToLabelValueMapping = slicer.util.getFirstNodeByClassByName("vtkMRMLColorTableNode", "CIP colors")
```

Comment:
I've noticed that CIP uses multiple different color tables. Rename `CIP colors` to `chest_region_colors_basic` (to indicate that it is derived from [chest_region_colors.ctbl](https://github.com/acil-bwh/SlicerCIP/blob/develop/Scripted/CIP_/CIP/ui/Resources/chest_region_colors.ctbl)). Change the name in LungCTAnalyzer, too, if needed (but hopefully we'll not need to define color tables there if we do the conversion to labelmap here).

If the color table is not needed for displaying the results then you can remove both the temporary labelmap and color node after the execution is completed, to not litter the scene with temporary nodes.